admin/base/views.py

```diff
@@ -5,14 +5,13 @@
 from django.utils.http import urlsafe_base64_decode
 from admin.common_auth.models import MyUser
 
-
 @login_required
 def home(request):
     context = {'user': request.user}
     return render(request, 'home.html', context)
 
 def password_reset_done(request, **kwargs):
-    messages.success(request, 'You have successfully reset your password and activated your admin account. Thank you')
+    messages.success(request, 'You have successfully reset your password and activated your admin account.')
     return redirect('auth:login')
 
 def password_reset_confirm_custom(request, **kwargs):
```

Comment:
I know this method isn't part of this PR but I can't figure out what it does differently than [the regular one](https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.views.password_reset_confirm)